recipe/meta.yaml

```diff
@@ -110,37 +110,18 @@ outputs:
         - m2w64-xz  # [win]
         - zlib
       run:
-        - curl
```

Comment:
@bgruening we don't necessarily have `run_exports` in all of our dependencies in conda-forge so we decided to keep the `run` as-is and iterate to remove them after the migration. In theory this explicit section should not change the results here BTW.

Reply:
I checked all the run_exports that I removed manually. Hope I got it correct.

Reply:
Cool! We decided to not use it at the moment b/c we did not want to check :smile: